backend/src/apiserver/client/subject_access_review.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2020 Arrikto Inc.  All Rights Reserved.
+
+package client
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	"k8s.io/client-go/rest"
+)
+
+func createSubjectAccessReviewClient() (v1.SubjectAccessReviewInterface, error) {
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to initialize kubernetes client.")
+	}
+
+	clientSet, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to initialize kubernetes client set.")
+	}
+	return clientSet.AuthorizationV1().SubjectAccessReviews(), nil
+}
+
+// CreateSubjectAccessReviewClientOrFatal creates a new SubjectAccessReview client.
+func CreateSubjectAccessReviewClientOrFatal(initConnectionTimeout time.Duration) v1.SubjectAccessReviewInterface {
+	var client v1.SubjectAccessReviewInterface
+	var err error
+	var operation = func() error {
+		client, err = createSubjectAccessReviewClient()
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
nit: this is equivalent to 

return err

Reply:
I blindly copied this part from the kubernetes core client :smile:. I'll do this fix in both places